### PR TITLE
[fix] Replace error assertion with soft warning 

### PR DESCRIPTION
**Pull request by mariogintili**
related to #178 and samselikoff/ember-cli-mirage#915

Unfamiliar if this actual solves anything or is compliant with the road the owners of this projects want to take but I need it as a workaround 😄 

Happy to change, close, etc based on request!

### CODE REVIEW

File: pretender.js [hunk 1]
```diff
@@ -261,9 +261,15 @@ function interceptor(pretender, nativeRequest) {
...
-          'Running two Pretender servers at once will lead to unexpected results!' +
-          'Please call .shutdown() on your instances when you no longer need them to respond.');
+    var debugMessage  = '[Pretender.js] You created a second Pretender instance while there was already one running. ';
+    debugMessage     += 'Running two Pretender servers at once will lead to unexpected results!';
+    debugMessage     += 'Please call .shutdown() on your instances when you no longer need them to respond.';
+
+    if (console.debug) {
+      console.debug(debugMessage);
+    } else {
+      console.log(debugMessage);
```
Comment (mariogintili) [inline pretender.js]: will probably need to change some tests for this

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pretender",
-  "version": "1.4.0",
+  "version": "1.4.1",
```
Comment (dustinfarris) [inline package.json]: Version bump should not be part of this PR.

File: pretender.js [hunk 3]
```diff
@@ -261,9 +261,15 @@ function interceptor(pretender, nativeRequest) {
   FakeRequest.prototype = proto;
 
   if (nativeRequest.prototype._passthroughCheck) {
-    throw new Error('You created a second Pretender instance while there was already one running. ' +
-          'Running two Pretender servers at once will lead to unexpected results!' +
-          'Please call .shutdown() on your instances when you no longer need them to respond.');
+    var debugMessage  = '[Pretender.js] You created a second Pretender instance while there was already one running. ';
```
Comment (dustinfarris) [inline pretender.js]: I would put a "DEPRECATED" in all caps here, e.g.

```js
console.warn('DEPRECATED: You created a second Pretender ...')
```

File: pretender.js [hunk 4]
```diff
@@ -261,9 +261,15 @@ function interceptor(pretender, nativeRequest) {
...
+    debugMessage     += 'Running two Pretender servers at once will lead to unexpected results!';
+    debugMessage     += 'Please call .shutdown() on your instances when you no longer need them to respond.';
```
Comment (dustinfarris) [inline pretender.js]: i think the style of the original change is better for concatenating a long message, e.g.:

```js
console.warn('DEPRECATED: You created ' +
  'a second Pretender instance ' +
  'while a second one ...');
```

File: pretender.js [hunk 5]
```diff
@@ -261,9 +261,15 @@ function interceptor(pretender, nativeRequest) {
...
+
+    if (console.debug) {
```
Comment (dustinfarris) [inline pretender.js]: Is this check necessary?

**Comment by trek** [comment]
This appears to be fixed on master. Thanks!

**Comment by mariogintili** [comment]
👍
